lighthouse-core/report/report-generator.js

```diff
@@ -56,7 +56,7 @@ class ReportGenerator {
 
     // Converts a name to a link.
     Handlebars.registerHelper('nameToLink', name => {
-      return name.toLowerCase().replace(/\s/, '-');
+      return name.toLowerCase().replace(/\s/g, '-');
```

Comment:
This is self-explanatory for me but just in case it isn't, say the `name` is `Progressive Web app`. Previously it became `progressive-web app` which is invalid.